src/test/playground-ide_test.ts

```diff
@@ -684,8 +684,9 @@ suite('playground-ide', () => {
       'playground-project'
     )) as PlaygroundProject;
 
-    await project.save();
-    await waitForIframeLoad(iframe);
+    const savePromise = project.save();
+    const reloadPromise = waitForIframeLoad(iframe);
+    await Promise.all([savePromise, reloadPromise]);
```

Comment:
Inlining seems a bit easier to read (fewer names to read):

```suggestion
    await Promise.all([waitForIframeLoad(iframe), project.save()]);
```